<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TV Denoising</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/program/cm.png</normaloff>:/program/cm.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolBox" name="optionsToolBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>160</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="optimPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>160</width>
       <height>500</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Optimization</string>
     </attribute>
     <widget class="QLineEdit" name="muEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0.2</string>
      </property>
     </widget>
     <widget class="QLabel" name="muLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>69</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>mu</string>
      </property>
     </widget>
     <widget class="QLabel" name="lambdaLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>69</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>lambda</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lambdaEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="niterSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>60</width>
        <height>27</height>
       </rect>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
     <widget class="QLabel" name="nIterLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Number iterations</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="linIterSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>60</width>
        <height>27</height>
       </rect>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="nCGLSLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Number CGLS steps</string>
      </property>
     </widget>
     <widget class="QLabel" name="epsLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Epsilon</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="epsEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>1e-3</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>353</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="text">
        <string>Total error</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="errorLcdNumber"/>
     </item>
     <item>
      <widget class="QLabel" name="constraintLabel">
       <property name="text">
        <string>Constraint violation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="constraintLcdNumber"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="iterButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>490</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Iterate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionReload"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
